lib/identity_cache/cached/primary_index.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module IdentityCache
+  module Cached
+    class PrimaryIndex
```

Comment:
You should update your commit message to mention this class name and not `PrimaryKey`.